data-model/developer-schema/annotation/0.1.0/schema/annotation-event.json

```diff
@@ -1,12 +1,12 @@
 {
   "$schema": "https://json-schema.org/draft/2020-12/schema",
-  "$id": "https://schemas.dissco.tech/schemas/annotations/0.1.0/annotation-event.json",
-  "description": "Object sent to annotation processing service from core backend. Internal use only.",
+  "$id": "https://schemas.dissco.tech/schemas/developer-schema/annotation/0.1.0/annotation-event.json",
```

Comment:
what is "deverloper-schema" here?

Reply:
That are schema's used for systems who want to interact to our system. In this case for example the MAS. The MAS doesn't needs to produces annotations which are slightly different than what the annotations endpoint produces. For example, the MAS does not need to indicate the `generator` part as this gets added by the annotation-processing-service. However, if the MAS supports batch, it does need to provide the batching information.